### DISCUSSION TITLE
RPG online multiplayer game

### DISCUSSION THREAD

**Discussion by lucasalmeida92** [mattjennings/svelte-phaser#18] [Q&A]
Hi there folks!
I wanna start developing a project that will be a Japanese course as a RPG game with interactions with other players (chat and missions)
I'm looking for a good and practical web solution because I don't need it to have lots of animations or other complex stuff. My only worry, in terms of performance, is about the multiplayer part.

Any thoughts if **svelte-phaser** can be a good solution?

I'm gonna make a svelte-phaser POC yet but any idea would be very welcome!

Thanks in advance

**Comment by mattjennings** [comment]
I'd love to see how your POC ends up! I would just caution that svelte-phaser is experimental and to my knowledge not used by many. There are still unknowns on best practices etc. However, these things can only be discovered by using it, so I'd still encourage you to try it and see how it goes!
